Artifacts/MsSql/Data/Security/2000-AdminApp-ClaimSets.sql

```diff
@@ -55,19 +60,42 @@ SELECT @actionId = ActionId FROM dbo.Actions WHERE ActionName = @actionName
 SELECT @claimSetId = ClaimSetId FROM dbo.ClaimSets WHERE ClaimSetName = @claimSetName
 
 PRINT 'Configuring Claims for Ed-Fi ODS Admin App Claimset...'
-INSERT INTO dbo.ClaimSetResourceClaims
-    (Action_ActionId, ClaimSet_ClaimSetId, ResourceClaim_ResourceClaimId, AuthorizationStrategyOverride_AuthorizationStrategyId)
-SELECT ActionId, @claimSetId, ResourceClaimId , @authorizationStrategyId FROM dbo.Actions a, @resourceClaimIds rc
-WHERE NOT EXISTS (SELECT 1
-                  FROM dbo.ClaimSetResourceClaims
-				  WHERE Action_ActionId = a.ActionId AND ClaimSet_ClaimSetId = @claimSetId AND ResourceClaimId = rc.ResourceClaimId)
+IF NOT EXISTS (SELECT 1
+                  FROM dbo.ClaimSetResourceClaimActions
```

Comment:
missing with condition AND ResourceClaimId = rc.ResourceClaimId in if not exists

Reply:
IF NOT EXISTS (SSELECT 1
                  FROM dbo.ClaimSetResourceClaimActions csraa,dbo.Actions a, @resourceClaimIds rc
				  WHERE csraa.ActionId = a.ActionId AND ClaimSetId = @claimSetId AND csraa.ResourceClaimId = rc.ResourceClaimId)